libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SmartSyncLogger.java

```diff
@@ -61,6 +70,20 @@ public static void e(String tag, String message, Throwable e) {
     }
 
     /**
+     * Logs an error log line.
+     *
+     * @param tag Log tag.
+     * @param origin Origin class.
+     * @param msg Log message.
+     * @param obj Object to be logged.
+     */
+    public static void e(String tag, Object origin, String msg, Object obj) {
```

Comment:
Do we need tag and origin?

SyncManagerLogger had a fixed tag ("SyncManagerLogger") and origin was used to differentiate log lines from different classes.

Also does SalesforceLogger print the component name anywhere in the log line?